<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Bt_Jump_Test">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>510</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Jump</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lb_UserName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>UserName</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_Login">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_ConnectToServer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="TextEdit_tcp_test_recv">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>201</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_UserName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_tcp_send_test">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_send">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_recv">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>recv</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_tcp_test_send">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="TextEdit_raw_recv">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_recv_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>recv_raw</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_UserPass">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="TextEdit_FSM_State">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>201</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
